### ISSUE TITLE
Increase contrast for better readability

### ISSUE THREAD

**Issue by nilshah98** [nilshah98/Portfolio#4]
**Issue :**
- Currently there is some text in orange, which does not have a high contrast in the light mode.
- Since it does not have high enough contrast it is hard to read, affecting the accessibility

**Reference :**
- 
![image](https://user-images.githubusercontent.com/22821480/133276231-78ac5d71-acaa-4922-bfda-4bcbff4d285f.png)
- [Accessibility contrast - web.dev](https://web.dev/color-contrast/?utm_source=lighthouse&utm_medium=lr)

**Acceptance Criteria :**
- [ ] Change the orange color in light mode to have better contrast and hence accessibility

**Comment by nilshah98** [comment]
Fixed with a541518404e1f2f14b4911108af1c5c13ab4c8d0
